pom.xml

```diff
@@ -112,7 +112,7 @@
         <dep.jettison.version>1.3.7</dep.jettison.version>
         <dep.jsonassert.version>1.2.3</dep.jsonassert.version>
         <dep.log4j.version>1.2.17</dep.log4j.version>
-        <dep.mongodb.version>3.3.0</dep.mongodb.version>
+        <dep.bson.version>3.6.3</dep.bson.version>
```

Comment:
the dependencies are ordered alphabetically. please stick to that order

Reply:
it seems to be a bug for my idea IDE

![image](https://user-images.githubusercontent.com/7821898/36599372-201bafd2-18ea-11e8-8416-aba309138a52.png)

it can't be both `a.csv` and `A.csv` at the same resources dir, may I change its name?

Reply:
first try to invalidate your idea caches. File -> Invalidate Caches
Probably this will help.

Reply:
the same

![image](https://user-images.githubusercontent.com/7821898/36599710-38b297bc-18eb-11e8-9c9d-4f551fae5320.png)

I think people who use idea may have same problem with this. I suggest to rename it `a`->`alpha` and `b`->`beta`. both file and sample define

Reply:
I just found why it may happend [here](https://confluence.jetbrains.com/display/IDEADEV/Filesystem+Case-Sensitivity+Mismatch). It's the matter of default system setting. I use MacOSX, it seems that this system is case-insensitive default